src/actions.py

```diff
@@ -145,6 +145,7 @@ def on_new_chat_member(bot: Bot, update: Update, job_queue: JobQueue):
                 "chat_id": chat_id,
                 "user_id": user_id,
                 "message_id": msg.message_id,
+                "timeout": timeout * 60,
```

Comment:
there are also `on_notify_timeout` jobs which also depend on the same `kick_timeout`.
Can you also reschedule them when the `kick_timeout` is changed?

Reply:
It's a good idea to keep the `if timeout >= 10` logic for them